commcare_export/writers.py

```diff
@@ -146,19 +146,29 @@ def write_table(self, table):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.book.save(self.file)
 
+
 class JValueTableWriter(TableWriter):
     """
     Write tables to JSON-friendly in-memory values
     """
 
     def __init__(self):
-        self.tables = []
```

Comment:
How come you changed it from a list to a dict? I guess this gets accessed by name somewhere?

Reply:
So previously each table would only get 'written' once. With the change to apply multiple `Emit` expressions for a single data source `write_table` function will get called once for every document so we need to be able to update existing tables.